docs/format/view-spec.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "View Spec"
```

Comment:
@samredai, is this extra file needed?

Reply:
Yes this is required. It adds the menu item to the left-nav on the docs site to link to this View Spec page which is built with the landing-page site.